docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,289 @@
+# Window
+
+A window contains the visible content of an application. When an application is created, it automatically creates a default window for building the main user interface of the application. You can also create additional windows as required to display additional content. A window also provides various events such as key event and touch event to the application.
+
+## Create window
+
+When you create an application, the default window is automatically created. The default window size is full screen:
+ ```csharp
+ MyApplication myApp = new MyApplication();
+ ```
+
+You can specify the initial size and position of the default window using the constructor of the application:
+```csharp
+public MyApplication(Size2D windowSize, Position2D windowPosition) : base(windowSize, windowPosition)
+{
+}
+
+MyApplication myApp = new MyApplication(new Size2D(1920, 1080), new Position2D(0, 0));
+```
+
+The instance of the default window can be retrieved:
+
+```csharp
+Window window = Window.Instance;
+```
+
+The window itself is just a blank container, and the views can be added to or removed from the window:
+
+```csharp
+// Create a new view
+View view = new View();
+
+// Add the view to the window
+window.Add(view);
+
+// Remove the view from the window
+window.Remove(view);
+```
+
+The following code shows how to add content to the window:
+
+```csharp
+// Get the window instance
+Window window = Window.Instance;
+window.BackgroundColor = Color.White;
+
+// Create a TextLabel
+TextLabel title = new TextLabel("Hello World");
+title.Size2D = new Size2D(window.Size.Width, window.Size.Height);
+title.HorizontalAlignment = HorizontalAlignment.Center;
+title.VerticalAlignment = VerticalAlignment.Center;
+
+// Add the text to the window
+window.Add(title);
+```
+
+![ NewWindow ](./media/new_window.png)
+
```

Comment:
Question. For now "Deleting Window" section is removed. 

So now in this page, we have description of how to create a window, but don't have description of how to remove a window.

I think updating "Delete Window" section is better than removing it. Would you please let me your opinion?

Reply:
API for deleting window is not public, so i think document for deleting window need to be added other ACR